modules/mri_violations/php/module.class.inc

```diff
@@ -85,9 +85,93 @@ class Module extends \Module
                     $user,
                     "Violated scan",
                     $DB,
-                    "SELECT COUNT(*) FROM mri_protocol_violated_scans
-                        LEFT JOIN candidate c USING (CandID)
-                    WHERE 1 = 1 ",
+                    "SELECT
```

Comment:
If this is the exact same query (and it's this long..) maybe it should go into a class constant and just be referred to in both places?

Reply:
@driusan I would say almost exact (but ignored some columns). I actually don't have any idea if there is any way to make the query small. It is the only way I could find make the count to be exact in Dashboard.

Reply:
but can you move it into a constant and just refer to the constant in both places?

Reply:
@driusan changes Done as per your suggestion. Let me know if it is all good now.

Reply:
Looks fine now, but it has conflicts and can't be merged.

Reply:
@driusan Oh, ok. just noticed that. I will fix it.

Reply:
@driusan Done. Fixed the conflicts.